openmdao/drivers/tests/test_doe_driver.py

```diff
@@ -1500,6 +1500,42 @@ def test_derivative_recording(self):
             for dv in ('x', 'y'):
                 self.assertEqual(derivs['f_xy', dv], expected_deriv['f_xy', dv])
 
+    def test_derivative_scaled_recording(self):
```

Comment:
Need the following decorator here, since this test requires `pyDOE2`  which is not always installed:
```
    @unittest.skipIf(pyDOE2, "only runs if 'pyDOE2' is not installed")
```

Reply:
Thanks for pointing that out. Should I also have added that to the other tests I added last time, `test_derivative_recording` and `test_derivative_no_recording`? They also use the `DOEDriver`, but I didn't put that decorator there.

Reply:
I see. Thank you!